facade/ignition/CVE-2021-3129.yaml

```diff
@@ -4,7 +4,10 @@ cve:       CVE-2021-3129
 branches:
     "master":
         time:     2020-11-17 09:18:00
-        versions: ['<=2.5.1', '>=2.0']
+        versions: ['<=2.5.1', '>=2.5.0']
```

Comment:
Looks like you need to swap upper and lower bound, I suggest to also use the `<` comparison operator to clarify the version in which this was fixed like this:

```
versions: [''>=2.5.0', '<2.5.2']
```

Reply:
Sorry I merged this already cause it does work as-is, but yeah I guess could be cleaned up. So @particleflux can you make a separate PR for that, sorry. You could also adjust the time entry on the 2.4 branch to Feb 18, 2021 to match the additional PR.

Reply:
Done, see: https://github.com/FriendsOfPHP/security-advisories/pull/546